initfiles/bash/etc/systemd/system/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set(esp "esp" "--daemon %i" "dafilesrv.service")
 set(roxie "roxie" "--topology=RoxieTopology.xml --logfile --restarts=2 --stdlog=0 --daemon %i" "dafilesrv.service")
 set(sasha "saserver" "--daemon %i" "dafilesrv.service")
 set(thor "thormaster_lcr" "--daemon %i MASTER=\${THORMASTER}:\${THORMASTERPORT}" "dafilesrv.service")
-set(thorslave "thorslave_lcr" "--daemon %i master=\${THORMASTER}:\${THORMASTERPORT} slave=.:\${SLAVEPORT} slavenum=\${SLAVENUM} logDir=${LOG_PATH}/\${THORNAME}" "dafilesrv.service")
+set(thorslave "thorslave_lcr" "--daemon %i --master=\${THORMASTER}:\${THORMASTERPORT} --slave=.:\${SLAVEPORT} --slavenum=\${SLAVENUM} --logDir=${LOG_PATH}/\${THORNAME}" "dafilesrv.service")
```

Comment:
These changes could target back to 7.8.x if we wanted to, but I don't think we plan on building an unsupported version.  This happened when we switched over from using mergeCmdParams() for globals, to loadArgsIntoConfiguration().  https://github.com/hpcc-systems/HPCC-Platform/commit/bd9e704545aa29ec05265b4c0363492c7f08ec22#diff-6095418795f3a03d23c0932453ccb5821ce8b11029717e8cdb861924cfe835a5

Tested this in a 7.8 build and it worked properly.